packages/slate-editor/src/extensions/autoformat/withAutoformat.ts

```diff
@@ -28,6 +29,12 @@ export function withAutoformat<T extends Editor>(editor: T, rules: AutoformatRul
             if (query && !query(editor, { ...rule, text })) continue;
 
             if (editor.selection) {
+                const [currentNode] = EditorCommands.getCurrentNodeEntry(editor) ?? [];
+
+                if (mode === 'block' && !isParagraphNode(currentNode)) {
+                    return;
+                }
```

Comment:
How about using bold and italic formatting shortcuts in Quotes, Lists, or Headings?

As I've said before, I don't believe this is a good fix.

Reply:
Alright. After reading the code more and testing more, I've realized that the intention was to keep using mark and text auto-formatters without limits. But there's a bug. It should be `continue` instead of `return` here.